views/car/edit.erb

```diff
@@ -0,0 +1,22 @@
+<h2>Edit Car</h2>
+
+<form action="/cars/<%= @car.id %>" method="post">
```

Comment:
This is true for a form that updates a car, although the prompt asked for a form that **CREATES** a car.